js/exercise_one/index.js

```diff
@@ -0,0 +1,144 @@
+function sortTableByCost(table) {
+  const tBody = table.tBodies[0];
+  const rows = Array.from(tBody.querySelectorAll("tr"));
+
+  const sortedRows = rows.sort((a, b) => {
+    const aCost = parseFloat(a.getElementsByTagName("td")[1].textContent.substring(1));
+    const bCost = parseFloat(b.getElementsByTagName("td")[1].textContent.substring(1));
+    return aCost - bCost;
+  });
+
+  while(tBody.firstChild){
+    tBody.removeChild(tBody.firstChild);
+  }
+  tBody.append(...sortedRows);
+}
+
+function chackAffordableItems(budget, table) {
+  const rows = table.getElementsByTagName('tr');
+  for (let i = 1; i < rows.length; i++) {
+    const row = rows[i];
+    const cells = row.getElementsByTagName('td');
+
+    if (cells.length >= 2) {
+      const cost = parseFloat(cells[1].textContent.substring(1));
+      const checkbox = cells[2].querySelector('input[type="checkbox"]');
+
+      if (cost <= budget) {
+        checkbox.checked = true;
+        budget -= cost;
+      } else {
+        checkbox.checked = false;
+      }
```

Comment:
You can rewrite this in two lines:
```suggestion
      if (cost <= budget) budget -= cost;
      checkbox.checked = (cost <= budget);
```

Reply:
I have a question regarding this comment. Does this make sense? What happens if:
Cost = 1
budget = 1
Then (cost <= budget) = true ==> budget -= cost ==> budget =0
Then checkbox.checked = (cost <= budget) = false ==> It will not be checked while it should have been checked

Reply:
You are right. You have two options, changing the order, or storing the comparison in its own variable and use it on both lines. 
It was just a suggestion. Also if you think is more readable like this, you can leave it like this.